### ISSUE TITLE
[HACKTOBERFEST] Add swag opportunity from GeeksforGeeks on daily problem practice

### ISSUE THREAD

**Issue by rahulkundu1209** [swapagarwal/swag-for-dev#1086]
## Wild swag opportunity appeared!

- [ yes] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : GeeksforGeeks

Q. Where have you seen the announcement (URL)?  
A. : https://practice.geeksforgeeks.org/problem-of-the-day/rule-book

Q. What kind of swag (clothing, stickers, ...)?  
A. : Clothing

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : Not Sure

Q. Does it ship worldwide?  
A. : Not Sure

- [ yes] I've checked that the promotion is legit.
- [yes ] I've checked that it's still available to the public.

**Comment by aslafy-z** [comment]
Duplicate of https://github.com/swapagarwal/swag-for-dev/issues/1014

**Comment by aslafy-z** [comment]
Feel free to work on #1014 @rahulkundu1209!

**Comment by rahulkundu1209** [comment]
@aslafy-z I have made the required changes but can you please assign me the issue under the hacktoberfest tag before I open a pull request.

**Comment by aslafy-z** [comment]
@rahulkundu1209 
The hacktoberfest label is added at the repository level, there is no need for it to be added to the pull request.
Please check your [hacktoberfest profile](https://hacktoberfest.com/profile/) once your pull request is merged, the PR should be marked as WAITING, which means the contribution was taken in account.
